docs/cms.rst

```diff
@@ -24,58 +24,326 @@ www.mozilla.org.
+   this step, the make file will also create a local admin user for you, using the
+   Mozilla LDAP email address you added in the previous step.
+#. Start bedrock running via ``npm start`` (for local dev) or ``make build run``
+   (for Docker).
+#. Go to ``http://localhost:8000/cms-admin/`` and you should see a button to login
+   with SSO. Click it and you should go through the OAuth flow and end up in the
+   Wagtail admin.
+
+Non-SSO authentication
+----------------------
+
+#. In your ``.env`` file set ``USE_SSO_AUTH=False``, and ``WAGTAIL_ENABLE_ADMIN=TRUE``.
+#. Run ``make preflight`` to update bedrock with the latest DB version.
+#. Create a local admin user with ``./manage.py createsuperuser``, setting both the
+   username, email and password to whatever you choose (note: these details will only
+   be stored locally on your device).
+#. Start bedrock running via ``npm start`` (for local dev) or ``make build run``
+   (for Docker).
+#. Go to ``http://localhost:8000/cms-admin/`` and you should see a form for logging in
+   with a username and password. Use the details you created in the previous step.
+
+Adding new content surfaces
+===========================
+
+This is an introduction to creating new content surfaces in the CMS. It is not a
+comprehensive guide, but rather a starting point to get you up and running with the
+basics.
+
+The page types that you see in the Wagtail admin are defined as regular models in
+Django. As such, you can define new page types in the same way you would define any
+other Django model, using Wagtail's field types and panels to define the data that
+can be entered into the page.
+
+When it comes to structuring CMS page models, there are some general guidelines to
+try and follow:
+
+- Models and templates should be defined in the same Django app that corresponds
+  to where the URL exists in bedrock’s AI hierarchy, similar to what we do for
+  regular Jinja templates already. For example, a Mozilla themed page should be
+  defined in ``/bedrock/mozorg/models.py``, and a Firefox themed page model should
+  be in ``/bedrock/firefox/models.py``.
+- Global models/blocks that are shared across many pages throughout the site should
+  be defined in ``/bedrock/cms/``.
+
+Structuring code in this way should hopefully help to keep things organized and
+migrations in a manageable state.
+
+Creating a new page model
+-------------------------
+
+Let’s start by creating a new Wagtail page model called ``TestPage``
+in ``bedrock/mozorg/models.py``.
+
+.. code-block:: python
+
+    from django.db import models
+
+    from wagtail.admin.panels import FieldPanel
+    from wagtail.fields import RichTextField
+
+    from bedrock.cms.models.base import AbstractBedrockCMSPage
+
+    class TestPage(AbstractBedrockCMSPage):
+        heading = models.CharField(max_length=255, blank=True)
+        body = RichTextField(
+            blank=True,
+            features=settings.WAGTAIL_RICHTEXT_FEATURES_FULL,
```

Comment:
Aside: I'm wondering about renaming `WAGTAIL_RICHTEXT_*` to `CMS_RICHTEXT_*` so that they don't look like they're core Wagtail settings. What do you think?